src/com/google/javascript/jscomp/CoverageInstrumentationPass.java

```diff
@@ -78,6 +79,13 @@ public void process(Node externsNode, Node rootNode) {
             compiler,
             rootNode,
             new BranchCoverageInstrumentationCallback(compiler, instrumentationData));
+      } else if (instrumentOption == InstrumentOption.ADVANCED) {
+        NodeTraversal.traverse(
+            compiler,
+            rootNode,
+            new AdvancedCoverageInstrumentationCallback(compiler, new HashMap<>()));
```

Comment:
Is passing in the Map as a parameter necessary? Looks like it's never used, and could just be initialized in the callback itself.

If not, prefer to use LinkedHashMap here, as it has deterministic iteration order